### ISSUE TITLE
docs-bug(all): High Contrast Mode homepage

### ISSUE THREAD

**Issue by twerske** [angular/components#23788]
#### Documentation Feedback

In Firefox High Contrast mode the documentation should be reviewed to ensure contrast ratios are maintained (see image)

![image (3)](https://user-images.githubusercontent.com/15061394/137986026-8ddd50f9-d13b-4def-a8de-785e35fe86ee.png)

**Comment by crisbeto** [comment]
This shouldn't be an issue anymore after we switched to using the media queries for high contrast.
